test/files/orm_columns.py

```diff
@@ -0,0 +1,42 @@
+from typing import List
```

Comment:
this gets tested with the plugin installed, so isnt really tested, right?

Reply:
yes, but the objective it to ensure that with the plugin nothing unsurprising happens.

I've looked at it in vscode and the types are inferred correctly